drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/htmlunit/HtmlUnitDriver.java

```diff
@@ -0,0 +1,24 @@
+package org.jboss.arquillian.drone.webdriver.htmlunit;
+
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.WebClientOptions;
+import org.openqa.selenium.Capabilities;
+
+public class HtmlUnitDriver extends org.openqa.selenium.htmlunit.HtmlUnitDriver {
+
+    public HtmlUnitDriver(Capabilities capabilities) {
+        super(capabilities);
+    }
+
+    @Override
+    public WebClient modifyWebClient(WebClient client) {
+        final WebClientOptions webClientOptions = client.getOptions();
+
+        return super.modifyWebClient(client);
```

Comment:
How should it work? It seems to me that superclass does [nothing](https://github.com/SeleniumHQ/htmlunit-driver/blob/15e9ecb98db58e6913cc781fa04ddb0903407626/src/main/java/org/openqa/selenium/htmlunit/HtmlUnitDriver.java#L488-L491)